playwright.config.js

```diff
@@ -72,11 +72,23 @@ export default defineConfig({
   projects: [
     {
       name: 'chromium',
-      use: { ...devices['Desktop Chrome'] }
+      use: {
+        ...devices['Desktop Chrome'],
+        viewport: {
+          width: 1920,
```

Comment:
are default viewports not sufficient for tests?

Reply:
no currently, with the default viewports, the sidebar nav will not show

Reply:
Oh, that's an interesting point. Do we have/need some tests that test in mobile/smaller viewports to verify that menus appear. Or maybe manual testing is good enough there (esp since there are a lot of random viewport sizings that would complicate Playwright testing).